### ISSUE TITLE
My RStudio can't find templates of ProjectTemplate templates in the New Project… wizard?

### ISSUE THREAD

**Issue by xiaoxi-david** [KentonWhite/ProjectTemplate#311]
My RStudio can't find any template of ProjectTemplate when I try to create a new project through the [New Project... wizard](https://rstudio.github.io/rstudio-extensions/rstudio_project_templates.html). 

I use ProjectTemplate 0.10.2 and RStudio 1.4.1717.

**Comment by KentonWhite** [comment]
The RStudio New Project...  wizard does not make a ProjectTemplate template (I would love if it did).  After creating your project in RStudio you will still need to create a ProjectTemplate template by running ProjectTemplate::create.project("MyProjectName")

Please let me know if that helps or if I'm not understanding the issue you are having!

**Comment by xiaoxi-david** [comment]
Ok, thank you. What a pity!

What prevents RStudio from detecting templates of ProjectTemplate? RStudio indeed can discover templates, for instance, the template of [ptexamples](https://github.com/rstudio/ptexamples).

**Comment by KentonWhite** [comment]
The Rstudio team would need to program that functionality into Rstudio.

**Comment by Hugovdberg** [comment]
I don't think that is actually the case. It looks to me from [this page](https://rstudio.github.io/rstudio-extensions/rstudio_project_templates.html) that the package could provide a `ProjectTemplate.dcf` that asks for the template name, and calls `ProjectTemplate::create_project_rstudio(...)` with the provided data that does some conversion to the `create.project` arguments and creates the template. A crude version of that dcf file is listed below, I left the `rstudio.project` argument out as that would be on by default using this method.

```
Binding: create_project_rstudio
Title: ProjectTemplate project
OpenFiles: eda.R

Parameter: template
Widget: TextInput
Label: Template name
Default: full
Position: left

Parameter: dump
Widget: CheckboxInput
Label: Dump ProjectTemplate files
Default: Off
Position: left

Parameter: merge.strategy
Widget: CheckboxInput
Label: Allow non-conflicting files in directory
Default: Off
Position: left
```

**Comment by KentonWhite** [comment]
Thanks @Hugovdberg   I'm not a big rstudio user.  Is this something that could be included in the project.template directory structure?  If so I'll add it for our rstudio users.
